Tests/TSCBasicTests/PathShimTests.swift

```diff
@@ -39,7 +39,7 @@ class WalkTests : XCTestCase {
         var expected: [AbsolutePath] = [
             "\(root)/usr",
             "\(root)/bin",
-            "\(root)/xbin"
+            "\(root)/etc"
```

Comment:
What's the test change for?

Reply:
I added this case years ago, back when some Android versions had this directory, matching `sbin/` below, but it looks like none do now. Updating this test to another random directory gets it to work again, and this is perfectly safe since it's scoped to just Android.